src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/service/SessionSummaryService.kt

```diff
@@ -26,15 +26,19 @@ class SessionSummaryService(
   private val inductionScheduleService: InductionScheduleService,
 ) {
 
-  val today: LocalDate = LocalDate.now()
```

Comment:
I initially thought this could be the cause of the bug, but it wasn't.
I could however be the cause of other bugs, because `today` is a class member and is only initialised when the bean is instantiated. Therefore, unless the bean is re-instantiated `today` will be wrong tomorrow! 🤣 

I moved it down into the method that uses it 👇